tests/unit/moduleapi/test_lazyfree.tcl

```diff
@@ -0,0 +1,27 @@
+set testmodule [file normalize tests/modules/test_lazyfree.so]
+
+start_server {tags {"modules"}} {
+    r module load $testmodule
+
+    test "modules allocated memory can be reclaimed in the background" {
+        set orig_mem [s used_memory]
+
+        # LAZYFREE_THRESHOLD is 64
+        for {set i 0} {$i < 100000} {incr i} {
+            r lazyfreelink.insert lazykey $i
+        }
```

Comment:
this is a long loop, that can take a while to execute.
if we need it that long, maybe convert it to use a deferring client (pipeline)

Reply:
Ok, changed to pipeline.